runtime/runtime.go

```diff
@@ -540,7 +542,16 @@ func (r *interpreterRuntime) parseAndCheckProgram(
 				sema.WithPredeclaredValues(valueDeclarations),
 				sema.WithPredeclaredTypes(typeDeclarations),
 				sema.WithValidTopLevelDeclarationsHandler(validTopLevelDeclarations),
-				sema.WithLocationHandler(runtimeInterface.ResolveLocation),
+				sema.WithLocationHandler(func(identifiers []Identifier, location Location) (res []ResolvedLocation) {
```

Comment:
Should this error be handled here or on the location where it is called?

Reply:
@janezpodhostnik Do you mean `Interpreter.LocationHandler` should have an error return value and this handler should just pass it down?

Reply:
Refactored it in d37c9f3